dataprofiler/profilers/profiler_options.py

```diff
@@ -761,3 +761,67 @@ def _validate_helper(self, variable_path='ProfilerOptions'):
             variable_path=variable_path + '.structured_options')
 
         return errors
+
+
+class TextProfilerOptions(BaseOption):
+    def __init__(self):
+        """
+        Constructs the TextProfilerOption object with default values.
+
+        :ivar is_case_sensitive: option set for case sensitivity.
+        :vartype is_case_sensitive: bool
+        :ivar stop_words: option set for stop words.
+        :vartype stop_words: Union[None, list(str)]
+        :ivar words: option set for word update.
+        :vartype words: BooleanOption
+        :ivar vocab: option set for vocab update.
+        :vartype vocab: BooleanOption
+        """
+        self.is_case_sensitive = True
+        self.stop_words = None
+        self.words = BooleanOption(is_enabled=True)
+        self.vocab = BooleanOption(is_enabled=True)
+
+    def is_prop_enabled(self, prop):
+        """
+        Checks to see if a property is enabled or not and returns boolean.
+
+        :param prop: The option to check if it is enabled
+        :type prop: String
+        :return: Whether or not the property is enabled
+        :rtype: Boolean
+        """
+        return BaseColumnOptions.is_prop_enabled(self, prop)
+
+    def _validate_helper(self, variable_path='TextProfilerOptions'):
+        """
+        Validates the options do not conflict and cause errors.
+
+        :param variable_path: current path to variable set.
+        :type variable_path: str
+        :return: list of errors (if raise_error is false)
+        :rtype: list(str)
+        """
+        errors = []
+        if not isinstance(self.is_case_sensitive, bool):
+            errors.append("{}.is_case_sensitive must be a Boolean."
+                          .format(variable_path))
+
+        if self.stop_words is not None and \
+                (not isinstance(self.stop_words, list)
+                 or len(self.stop_words) < 1
```

Comment:
Could the user ever want no stop words? This prohibits them from doing so but is there a use case for this?

Reply:
makes sense. I removed that check. The users can still set empty list if they want.